meta/contributor-handbook.md

```diff
@@ -0,0 +1,88 @@
+
+For the first pattern you contribute, you should aim for maturity levels `1: initial` or `2: structured`. 
+
+If you want to help promoting a pattern one maturity level up, we suggest to first create an issue to discuss the matter and see if someone else is working on that already. After that, you can create a pull request with the necessary changes.
+
+To achieve a given maturity level a pattern has to satisfy the requirements for that given maturity level and lower levels. The following sections lay out the requirements per maturity level in detail.
+
+
+### Requirements: Level 1 - Initial
+
+Patterns (or other documents) of level 1 are stored in the directory `/1-initial`.
+
+- Document types:
+	- Patterns (`pattern-*.md`)
+	- Donuts - patterns without solution or other missing sections (`donut-*.md`)
+	- Tale - unstructured information in any form (`tale-*.md`)
+	
+- Validation requirements:
+	- N/A
+	
+- Content requirements:
+	- The document is readible & comprehensible for other's than the author(s)
+	- The author(s) contribute the contents according to the [license](../LICENSE.txt) & are allowed to do so
+	- Thoughts and contents by third parties are quoted / referenced explicitly
+
+	
+### Requirements: Level 2 - Structured
+
+Patterns of level 2 are stored in the directory `/2-structured`.
+
+- Document types:
+	- Patterns (`pattern-*.md`)
+	
+- Validation requirements:
+	- Is validated by at least one known instance 
+		- Alternatively: key elements of the pattern have been validated in separate contexts and, in consequence, it is justified to believe the full solution will function
+	
+- Content requirements:
+	- Complies with the [patterns format](pattern-template.md)
+	- Complies with *basic style guide*(#) - *Oops! We have not yet developed this*
+	
+	
+### Requirements: Level 3 - Agreed
+
+Patterns of level 3 are stored in the directory `/3-agreed`.
+
+- Document types:
+	- Patterns (`pattern-*.md`)
+	
+- Validation requirements:
+	- Is validated by at least three known instances
+	- Considers all known instances to the best of working group members' knowledge
+	- Community agreement (via lazy consensus of trusted committers) on correctness of contents
+	
+- Content requirements:
+	- Uses & has no conflicts with working group terminology (defined by glossary / implicit usage) - *Oops! We have not yet developed this.*
+	- Fits & has no conflicts with existing patterns (of this maturity level or higher)
+	- Thorough review by at least two trusted committers
+	
+	
+### Requirements: Level 4 - Established
+
+Patterns of level 4 are stored in the directory `/4-established`.
+
+- Document types:
+	- Patterns (`pattern-*.md`)
+	
+- Validation requirements:
+	- Complies with *advanced style guide*(#) - *Oops! We have not yet developed this.*
+	- Community agreement (via lazy consensus of trusted committers) this representing universal truth about InnerSource (using best judgment) and it being responsible to push it out

```

Comment:
I wonder which additional quality improvement over stage 3 this requirement would achieve? The statement "universal truth" is too easy to argue about, and in stage 3 we already achieve "Community agreement (via lazy consensus of trusted committers) on correctness of contents".

So I would argue that we can leave this bullet point away completely.

Reply:
```suggestion
```
I agree. It's to vague.